src/swap/SwapReviewScreen.test.tsx

```diff
@@ -152,25 +154,80 @@ describe('SwapReviewScreen', () => {
       // Swap To
       expect(getByTestId('ToSwapAmountToken')).toHaveTextContent('3.10 cUSD')
       // Exchange Rate
-      expect(getByText('1 CELO ≈ 3.10 cUSD')).toBeTruthy()
+      expect(getByTestId('ExchangeRate')).toHaveTextContent('1 CELO ≈ 3.10 cUSD')
       // Estimated Gas
       expect(getByTestId('EstimatedGas')).toHaveTextContent('0.00015 CELO')
       // Swap Fee
       expect(getByTestId('SwapFee')).toHaveTextContent('swapReviewScreen.free')
     })
   })
 
+  it('should display correct exchange rate when buyAmount is used', async () => {
+    const newStore = {
+      ...store,
+      swap: {
+        ...store.swap,
+        swapUserInput: {
+          ...store.swap.swapUserInput,
+          toToken: mockCeloAddress,
+          fromToken: mockCusdAddress,
+          swapAmount: {
+            FROM: '3100000000000000000',
+            TO: '1000000000000000000',
+          },
+          updatedField: Field.TO,
+        },
+      },
+    }
+
+    const newMockStore = createMockStore(newStore)
+
+    mockFetch.mockResponse(
+      JSON.stringify({
+        unvalidatedSwapTransaction: {
+          sellToken: mockCusdAddress,
+          buyToken: mockCeloAddress,
+          buyAmount: '1000000000000000000',
+          sellAmount: '3100000000000000000',
+          price: '3.1',
+          gas: '300000',
+          gasPrice: '500000000',
+        },
+      })
+    )
+
+    const { getByTestId } = render(
+      <Provider store={newMockStore}>
+        <SwapReviewScreen />
+      </Provider>
+    )
+
+    await waitFor(() => {
+      // Swap From
+      expect(getByTestId('FromSwapAmountToken')).toHaveTextContent('3.10 cUSD')
+      expect(getByTestId('FromSwapAmountTokenLocal')).toHaveTextContent('$3.10')
+      // Swap To
+      expect(getByTestId('ToSwapAmountToken')).toHaveTextContent('1.00 CELO')
+      // Exchange Rate
+      expect(getByTestId('ExchangeRate')).toHaveTextContent('3.10 cUSD ≈ 1 CELO')
```

Comment:
I wonder what's best to display for our users? 
`3.10 cUSD ≈ 1 CELO` or `1 cUSD ≈ 0.32 CELO`

Reply:
`3.10 cUSD ≈ 1 CELO` was the easiest with our current implementation so I went with that.